src/cheri/insns/jalr_32bit.adoc

```diff
@@ -16,56 +16,30 @@ include::base_isa_extension.adoc[]
 
 #Begin changed since last ARC review#
 
+
 Description::
 Indirect jump to the target capability in `{cs1}` with an address offset.
 +
+NOTE: The  description below contains three hooks for extending <<JALR_CHERI>> behavior, used by extensions and/or encoding formats to give a large degree of extensibility.
+ Unless an extension and/or encoding format, such as <<section_rvy_sentry_insn_ext>>,  is implemented which explicitly reference the hooks, then take no action for any of them.
```

Comment:
```suggestion
NOTE: The  description below contains three hooks for extending <<JALR_CHERI>> behavior, used by extensions to give a large degree of extensibility.
 Unless an extension, such as <<section_rvy_sentry_insn_ext>>, is implemented that explicitly reference the hooks, then take no action for any of them.
```

I thought the conclusion was to avoid mentioning encodings here, since we are only discussing semantics here. @nwf does the suggested wording sound good to you?

Reply:
it's good for me

Reply:
A challenge to that is that the default encoding chapters also include semantics, so it's hard to talk about the one without the other.